adam/experiment/log_experiment.py

```diff
@@ -348,22 +348,26 @@ def learner_factory_from_params(
                 smoothing_parameter=0.002,
```

Comment:
I thought I did make configurable for the simulated experiments. I may have left it as unconfigurable for the symbolics since it doesn't do anything there.

Reply:
Ah you're correct! I missed these were the symbolic matchers 🙃 apologies.

Reply:
No worries.